Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Beckn/Search.hs

```diff
@@ -165,58 +159,45 @@ handler merchantId sReq = do
           for_ listOfSpecialZoneQuotes QQuoteSpecialZone.create
         return (Just (mkQuoteInfo fromLocation toLocation now <$> listOfSpecialZoneQuotes), Nothing)
       else do
+        driverPoolCfg <- getDriverPoolConfig merchantId result.distance
         estimates <-
           case org.farePolicyType of
             SLAB -> do
```

Comment:
This two cases duplicated a lot. I think you can create some common function to avoid duplication, like you already  created `farePolicyHelper`